design/src/genre-analysis.md

```diff
@@ -0,0 +1,121 @@
+# Genre Analysis: Factory Builders
+
+## Defining Games
+
+- Factorio
+  - colonize an alien planet and launch a rocket
+- Satisfactory
+  - open world first person factory building
+- Shapez.io
+  - abstract factory puzzles
+- Dyson Sphere Program
+  - gather resources from an entire solar system
+
+## Defining characteristics
+
+- automated resource extraction
+- automation resource refinement
+- micro-optimizations
+- gated technology progression
+
+## Adjacent Genres
+
+- Base and city builders
+  - Build out a complex and delightful area, but with no / little automation
+- Zachlike puzzles
+  - Micro-optimization as an entire game
+- Colony sims
+  - Balance complex systems, but with a focus on the needs of your workers, not scaling production
+- Idle games
+  - Automate everything, but without the complex chains
+- Train simulators
+  - Focusing almost entirely on logistic networks powered by trains
+
+## Blended Genre Games
+
+- Mindustry (tower defense)
+- Hydroneer | Astroneeer | Atrio (exploration)
+- Automachef (Zachlike + cooking game)
+- Infinifactory (Zachlike)
+- Timberborn (city builder)
+
+## Progression Loop
+
+- explore
+- find resource patch
+- exploit resource
+- refine resource
+- build into science
+- unlock new technology
+- optimize base with new technology
+- explore to find new resource type to exploit
+
+## Sources of Challenge
+
+- player-driven optimization
+- resource refinement puzzles
+- logistics layout
+- (optional) combat disrupting operations
+- (optional) temporal variation creating unstable
+
+## Design Tensions
+
+- complexity vs meaningful richness
+- helpful innovations vs trivializing the game
+- disruption forces interesting changes vs frustration and sadness to see your stuff get wrecked
+- long play time vs tedious waiting around
+- player fantasy of being in the game vs annoyance of inventory space and walking times
+- giant automated bases vs computational limits
+
+## Standard Mechanics
+
+- technology
+  - process and spend resources to unlock new options
+- blueprints + copy-paste
+- assembling machines
+  - select a recipe for what you want to make
+  - often specialized variants
+- resource extraction
+  - usually but not always spatially limited
+  - often but not always non-renewable
+- raw materials, intermediates and end products
+- electricity
+- conveyer belts
+  - moves items from place to place
+- inserters
+  - loads items to and from assembling machines and chests
+- chests
+  - stores goods
+- (optional) bots
+  - takes goods from source to destination in a smarter but more expensive way
+- (optional) trains
+  - moves huge amounts of goods at once, but with expensive infrastructure and large batches
+
+## Common Problems
+
+- mediocre combat
+  - poorly integrated
+  - frustrating
+  - snowbally
+  - thematically incoherent
+  - difficult to balance
+- poorly managed complexity in UX
+- poor tutorialization
```

Comment:
Zachlikes being a relevant exception in the adjacent genre.

Reply:
Yep!